go/vt/vtctl/reparentutil/durability.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Vitess Authors.
+Copyright 2022 The Vitess Authors.
```

Comment:
Until we decide what to do about the copyright, we should not be making any changes.
Technically, it is incorrect to change the copyright year.

Reply:
This is an unfortunate incorrect diff from GitHub...
I actually renamed the `durability.go` file to `durability_policies.go` since that file had the implementations of the durability policies.
Then later I add `durability.go` for having functions with respect to durability like `Revoked`.

So the file `durability_policies.go` has the year 2021, since it is unchanged except for rename. And `durability.go` now holds new code all introduced in this year.